src/Aspire.Hosting/DistributedApplicationBuilder.cs

```diff
@@ -278,6 +278,35 @@ public DistributedApplicationBuilder(DistributedApplicationOptions options)
 
     private void ConfigureHealthChecks()
     {
+        _innerBuilder.Services.AddSingleton<IValidateOptions<HealthCheckServiceOptions>>(sp =>
```

Comment:
Do we log from the other configure options code?

Reply:
No ... we don't log in other places. However the reason I did it here is that I felt that if we just gave a generic error that a health check was missing it wouldn't be very helpful to the developer. By outputting to the logs we make it easier to find.

Happy to remove this though because this check is pretty aggressive and should show up for folks as soon as they make the mistake.